sourcewrap/blank.go

```diff
@@ -10,11 +10,15 @@ import (
 )
 
 // Blank operates as a blank source in its default state, but has a SetSource
-// method for setting the source later, it then uses the dials.Watcher
-// interface to update the View it's inserted into.
+// method for setting the source later, it then uses the dials.Watcher interface
+// to update the View it's inserted into.
 //
 // Blanks cannot be reused as they have to be aware of parameters of a
 // particular View.
+//
+// Note that when using this, consider setting the `SkipInitialVerification` to
+// `true` if any data from Blank-wrapped sources is expected to be considered in
+// a configuration that implements `dials.VerifiedConfig`.
```

Comment:
I think this is slightly confusing.

Expecting data from the blank-wrapped sources to be "considered" isn't a problem. The important determination is if data from the wrapped source is necessary for the state of the config to pass `Verify()`.